### PR TITLE
Cypress reorganization

### PR DESCRIPTION
**Pull request by olemartinorg**
## Description
When running cypress tests entirely locally, it is only possible to have one app running at a time, but the cypress tests were mixing different apps in the same tests. I split these tests into different directories, and made it a tiny bit easier to run these tests on Linux.

## Verification
- [X] **Your** code builds clean without any errors or warnings
- [X] Manual testing done (required)
- [X] Relevant automated test added (if you find this hard, leave it and we'll help out)
- [x] All tests run green

## Documentation
- ~~[ ] User documentation is updated with a separate linked PR in [altinn-studio-docs.](https://github.com/Altinn/altinn-studio-docs) (if applicable)~~
- ~~[ ] Changelog is updated with a separate linked PR~~
  - ~~[ ] [altinn-app-frontend](https://docs.altinn.studio/community/changelog/app-frontend/)- (if applicable)~~
  - ~~[ ] [nuget-packages](https://docs.altinn.studio/community/changelog/app-nuget/) - (if applicable)~~

### CODE REVIEW

**Comment by SandGrainOne** [comment]
Could somone verify that there are no Azure DevOps pipelines affected by this change, or ensure me that it's being handled?

File: .github/workflows/cypress-altinn-app-frontend.yml [hunk 1]
```diff
@@ -55,13 +55,10 @@ jobs:
           export CYPRESS_PROJECT_ID=y2jhp6
           export CYPRESS_RECORD_KEY=${{ secrets.CYPRESS_RECORD_KEY }}
           export GITHUB_TOKEN=${{ secrets.GITHUB_TOKEN }}
-          ./node_modules/.bin/start-server-and-test 'cd $GITHUB_WORKSPACE/src/altinn-app-frontend && yarn start' http://localhost:8080/altinn-app-frontend.js 'yarn run test:appfrontend:headless --env environment=tt02,testUserName=tt02testuser,testUserPwd=${{ secrets.CYPRESS_ALTINN_USERPWD }} --record --parallel --tag "altinn-app-frontend" --group altinn-app-frontend --ci-build-id $GITHUB_RUN_ID-$GITHUB_RUN_ATTEMPT'
+          ./node_modules/.bin/start-server-and-test 'cd $GITHUB_WORKSPACE/src/altinn-app-frontend && yarn start' http://localhost:8080/altinn-app-frontend.js 'yarn run test:all:headless --env environment=tt02,testUserName=tt02testuser,testUserPwd=${{ secrets.CYPRESS_ALTINN_USERPWD }} --record --parallel --tag "altinn-app-frontend" --group altinn-app-frontend --ci-build-id $GITHUB_RUN_ID-$GITHUB_RUN_ATTEMPT'
         working-directory: test/cypress
 
   altinn-app-frontend-test-on-fork-pr:
-    if: |
```
Comment (jeevananthank) [inline .github/workflows/cypress-altinn-app-frontend.yml:62]: removing this has run the test twice. refer [this](https://github.com/Altinn/app-frontend-react/runs/6271678922?check_suite_focus=true).

Comment (olemartinorg) [reply .github/workflows/cypress-altinn-app-frontend.yml:62]: Yikes! I didn't even mean to remove that! Thanks @jeevananthank! I tried using using `act` to test the workflow locally and I had to remove this check to get it to work. Good thing we do code reviews!

**Comment by olemartinorg** [comment]
@SandGrainOne Maybe @haakemon knows best?

**Comment by haakemon** [comment]
> Could somone verify that there are no Azure DevOps pipelines affected by this change, or ensure me that it's being handled?

We are no longer using Azure DevOps pipelines for this repo. Everything is run as Github Actions, and they all pass, so I think we are good 😁

**Review by haakemon — APPROVED** [review]
Nice!

**Pull request merged**